src/data/backup.ts

```diff
@@ -340,7 +340,7 @@ export const computeBackupType = (
   if (backup.with_automatic_settings) {
     return "automatic";
   }
-  if (isHassio && backup.extra_metadata?.["supervisor.addon_update"]) {
+  if (isHassio && backup.extra_metadata?.["supervisor.addon_update"] != null) {
```

Comment:
I'd rather just check if the key is there honestly, or check it if truthy like before, but it is ok.

Reply:
Yeah but it would work if `supervisor.addon_update: null` for some reason.